src/wtc-gallery-component.js

```diff
@@ -458,10 +460,14 @@ class Gallery extends ElementController {
     this.currentIndex = +next.dataset.index;
 
     if (!this.options.loop && this.options.nav) {
+      console.log(this.currentIndex);
```

Comment:
Should we be leaving a `console.log()` in here? Other than that, this is awesome man. Thanks!